spec/jsSpecs/common-spec.js

```diff
@@ -106,9 +109,20 @@ commonSpec.makeCmd = function(cmd) {
     return `node ${path.join('bin', cmd + '.js')}`;
 };
 
+commonSpec.createOutDir = function(filePath, mode) {
+    if (!fs.existsSync(filePath)) {
+        shelljs.mkdir(filePath);
+
+        if (mode) {
```

Comment:
I think if (mode)~ is moved to out of if(!fs.existSync(filePath)).
If the filePath is already exist and mode is not matched required mode, this case is not support to intended functionality correctly.

Reply:
I agree with you. I'll modify it.